tests/test_db.py

```diff
@@ -64,6 +66,34 @@ class _Object(object):
     pass
 
 
+class _RefOperations(object):
```

Comment:
Please add a comment for this class explaining what it's for.

You might also want to move it down closer to where it's used?

Reply:
Done